.gitmodules

```diff
@@ -77,4 +77,4 @@
 	url = https://github.com/open-telemetry/opentelemetry-cpp.git
 [submodule "src/nvmeof/gateway"]
 	path = src/nvmeof/gateway
```

Comment:
BTW, as `ceph` is quite redundant in this context, I think this would be enough (was there any specific reason to create another subdir?):
```suggestion
	path = src/nvmeof
```

Reply:
Understood. Make sense! Thanks for clarifying.